dartdoc_test/lib/src/analyzer.dart

```diff
@@ -18,31 +18,31 @@ import 'package:analyzer/error/error.dart';
 import 'package:dartdoc_test/src/resource.dart';
 import 'package:source_span/source_span.dart';
 
+import 'logger.dart';
+
 void getAnalysisResult(
   AnalysisContext context,
   CodeSampleFile file,
 ) async {
   final result = await context.currentSession.getErrors(file.path);
   if (result is ErrorsResult) {
+    if (result.errors.isEmpty) {
+      log('No issues found!');
+      return;
+    }
     for (final e in result.errors) {
-      if (e.errorCode.type != ErrorType.SYNTACTIC_ERROR &&
-          e.errorCode.type != ErrorType.COMPILE_TIME_ERROR) {
-        continue;
-      }
```

Comment:
Why were we doing this before?

Reply:
I was trying to figure out how to hide the import errors (`this error is caused by generated code.`) before adding the verbose flag. But this would have ignored the necessary errors, so I removed it.

Reply:
This is unrelated to this change, but you could consider looking at:

https://pub.dev/documentation/analyzer/5.13.0/error_error/ErrorCode-class.html

There are codes for every error code, see: https://dart.dev/tools/diagnostic-messages#unused_import

So probably if `e.errorCode.uniqueName == 'unused_import'` you can just ignore it.
Or maybe it's `e.errorCode.name == 'unused_import'` (you can probably just try to print them to figure out which is which :rofl: 

But given how we generate sample code, maybe it's fine to always ignore "unused import" issues.